vnext/Microsoft.ReactNative/CompositionSwitcher.idl

```diff
@@ -110,6 +110,17 @@ namespace Microsoft.ReactNative.Composition
     void Color(Windows.UI.Color color);
   }
 
+  [webhosthidden]
+  [experimental]
+  interface ISwitchThumbVisual
+  {
+    IVisual InnerVisual { get; };
+    void Size(Windows.Foundation.Numerics.Vector2 size);
+    void Position(Windows.Foundation.Numerics.Vector2 position);
+    Boolean IsVisible { get; set; };
+    void Color(Windows.UI.Color color);
+  }
```

Comment:
Is there a write up anywhere about why/when you need to create these visual wrappers? (as opposed to talking to a generic visual directly) If not, maybe a MD file in the Fabric/Composition directory that describes how to approach this? Is this to enable compositor abstraction (system/lifted/airspace)?

Reply:
I have a write-up in the internal One-Note, but's it's very minimal. Once it's filled out more, I can add it as a MD file! :)